src/components/modal/veiledergruppe/veileder-gruppe-modal-lage.tsx

```diff
@@ -61,6 +67,13 @@ function VeilederGruppeModalLage(props: VeilederGruppeModalProps & Omit<ModalPro
         setGruppeNavn(props.lagretFilter ? props.lagretFilter.filterNavn : '');
     }, [props.lagretFilter]);
 
+    useEffect(() => {
+        for (let i = 0; i < lagretFilterState.data.length; i++) {
```

Comment:
inga for-lopar tack ;)  hva gör denna koden??

Reply:
porque no? den lager en liste med gruppenavn som den itererer gjennom senere når den skal sjekke at det nye gruppenavnet ikke finnes fra før. om det var forståelig

Reply:
er sikkert ikke beste metoden, but it works! forslag til endring, eventuelt?

Reply:
const veiledergruppeListe = lagretFilterState.filter(filter => filter.filterNavn)

Reply:
men hvorför er det i en useEffect?

Reply:
fordi den skal sjekke listen som ligger der over veiledergrupper før den går inn i rediger/lage ny-modalen, altså før den gjør noe annet. på denne måten lagrer den listen i en konstant, så den kan hentes ut når det trengs. med 
`const veiledergruppeListe = lagretFilterState.map(filter => filter.filterNavn)`
får jeg feilmeldingen: 
`TS2339: Property 'map' does not exist on type 'LagretFilterState'.`